shared/js/project.js

```diff
@@ -316,6 +316,8 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skinId === 'hoc2015' || appOptions.skinId === 'infinity') {
```

Comment:
The most complete fix would be to have a mapping of `appOptions.app` and `appOptions.skinId` that correspond to standalone projects (since both of these are needed when checking if a project template is equivalent).  Are there other places you can think of where we need to remove the 'Remix' button?